src/encoded/static/scss/encoded/modules/_cart.scss

```diff
@@ -1267,3 +1260,41 @@ $file-type-colors: (
         width: auto;
     }
 }
+
+// Contents of the type-selection modal for /cart-search/ page, Report view control button.
+.cart-type-selector {
```

Comment:
Where is "cart-type-selector"  used? I search for it and only saw one reference in this css file.

Reply:
Yeah, I think I stopped using it at some point.